sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Diagnostics/BlobCheckpointStoreInternal.Diagnostics.cs

```diff
@@ -95,19 +95,19 @@ static BlobCheckpointStoreInternal()
         /// <param name="eventHubName">The name of the specific Event Hub the checkpoint is associated with, relative to the Event Hubs namespace that contains it.</param>
         /// <param name="consumerGroup">The name of the consumer group the checkpoint is associated with.</param>
         /// <param name="clientIdentifier">The unique identifier of the client that authored this checkpoint.</param>
-        /// <param name="sequenceNumber">The sequence number associated with the checkpoint.</param>
-        /// <param name="replicationSegment">The replication segment associated with the checkpoint.</param>
-        /// <param name="offset">The offset associated with the checkpoint.</param>
+        /// <param name="sequenceNumber">The sequence number associated with the checkpoint. This value is set to <c>-1</c> if one is not provided.</param>
```

Comment:
These could also be defaulted to long.MinValue. My rationale for changing to -1 is that when looking at the logs the long.MinValue looks somewhat random and seems like it's coming from somewhere, not that it's just nothing was provided.